shared/constants/tracker.js

```diff
@@ -36,6 +39,7 @@ export const updateProofState = 'tracker:updateProofState'
 export const reportLastTrack = 'tracker:reportLastTrack'
 
 export const onClose = 'tracker:onClose'
+export type OnClose = TypedAction<'tracker:onClose', void, void>
```

Comment:
You can not, flow doesn't want to evaluate the js (event if its a simple const string). Maybe a future flow version will fix that but for now you have to pass in the literal string if you want it to match the literal string as the type